packages/react-components/src/components/AutoComplete/types.ts

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import { Placement, Strategy } from '@floating-ui/react';
+
+import { InputProps } from '../Input';
+import { IPickerListItem } from '../Picker';
+
+// selectedItemBody is unnecessary for AutoCompleteListItem, key should be === name
+export type IAutoCompleteListItem = Omit<
```

Comment:
What if the names would be duplicated in the options? Key attributes should be unique.

Reply:
for this component's purposes they shouldn't be - it'd be impossible to differentiate between two options and typing in their name would show both of them